### PR TITLE
Union of two arrays

### PR DESCRIPTION
**Pull request by soumikbaithalu**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes 

### Description

Sort both arrays then iterators for arr1, arr2 then go along both arrays if an element is lower than another, increment iterator in that array, else part if we find equal elements add element to the result and increment iterators of both arrays

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

This issue related to #4960

### CODE REVIEW

File: Python/other/Union_of_two_arrays.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+
+if __name__ == "__main__":
+    arr1 = list(map(int,input("Enter the Number in list 1:\n").split()))
+    arr2 = list(map(int,input("Enter the Number in list 2:\n").split()))
+    print(union_of_arrays(arr1, arr2))
+
+'''
+Time Complexity : O((n+m)log(n+m))
+Space Complexity : O(n+m)
```
Comment (ankitaggarwal23) [inline Python/other/Union_of_two_arrays.py:37]: Add a new line after SC

File: Python/other/Union_of_two_arrays.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+INPUT:
+    Enter the list 1: 
+    1 2 3 2 0
+	Enter the list 2: 
+    5 1 2 7 3 2
+
+OUTPUT  
+    [1, 2, 2, 3]
```
Comment (ankitaggarwal23) [inline Python/other/Union_of_two_arrays.py:46]: Remove the tab space and align properly

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top
- Use proper code Format 
- Remove extra lines of code and spaces

File: Python/other/Union_of_two_arrays.py (hunk 2 above)
Comment (soumikbaithalu) [reply Python/other/Union_of_two_arrays.py:46]: I have modify the file as you request @ankitaggarwal23

**Review by siddharth25pandey — APPROVED** [review]
@soumik2012 LGTM 💯 Able to merge

**Comment by siddharth25pandey** [comment]
@all-contributors please add @soumik2012 for his code

**Pull request merged**